mcan/src/bus.rs

```diff
@@ -157,13 +157,26 @@ pub trait DynAux {
     /// disable CAN operation.
     fn initialization_mode(&self);
 
-    /// Sets MCAN up for being shut down.
+    /// Requests the peripheral to enter "power down" mode.
+    ///
     /// See `Power Down (sleep mode)`, in the peripheral docs, for user
     /// consideration regarding clocking and message handling.
-    fn shutdown(&self);
+    ///
+    /// It is also worth noting that this mode should not be interpreted as the
+    /// peripheral being "powered off", since it is still possible to
+    /// configure the peripheral while in this mode.
+    fn power_down_mode(&self);
 
-    /// Check if csa bit is set
-    fn ready_for_shutdown(&self) -> bool;
+    /// Check if the transition to `Power Down (sleep mode)` is complete and
+    /// that it is safe to completely disable the peripheral.
+    ///
+    /// Returns `false` if the peripheral is still handling outgoing or
+    /// incomming messages.
+    ///
+    /// If the bus is heavily congested, the peripheral
```

Comment:
Nit: Look over the formatting on this paragraph, it seems to be wrapped prematurely.

Reply:
This happend due to the fact that I used `rustfmt` with nightly.

Default then becomes 80 characters.
https://rust-lang.github.io/rustfmt/?version=v1.7.0&search=#comment_width

I'll revert to stable.

Reply:
Alright, it was just me making a mistake.  :smile: 
80 is the default length for the other wrapped comments in the repo.

Reply:
Fixed.